testrunner-running/src/main/scala/edu/illinois/cs/testrunner/execution/TestListener.java

```diff
@@ -35,7 +39,23 @@ public void testIgnored(Description description) throws Exception {
 
     @Override
     public void testStarted(Description description) throws Exception {
-        times.put(JUnitTestRunner.fullName(description), System.nanoTime());
+        String fullTestName = JUnitTestRunner.fullName(description);
+        times.put(fullTestName, System.nanoTime());
+
+        String phase = Configuration.config().getProperty("statecapture.phase", "");
+        String state = Configuration.config().getProperty("statecapture.state", "");
+        if (Configuration.config().getProperty("replay.dtname").equals(fullTestName)) {
+            if (phase.equals("capture_before")) {
+                StateCapture sc = new StateCapture(fullTestName);
+                System.out.println("MainAgent.targetTestName: " + Configuration.config().getProperty("replay.dtname") +
```

Comment:
This print statement is somewhat redundant considering that the two names must be equal if they end up in this branch.